test/tcp-reconstruction/src/main.rs

```diff
@@ -1,5 +1,4 @@
-#![feature(box_syntax)]
-#![feature(asm)]
+#![feature(box_syntax, asm)]
```

Comment:
We don't need feature flags more than once.

Reply:
Same as my comment in the other PR. I prefer have them as separate lines. Easier to add/remove and identify what features are enabled.

Reply:
i do not feel strongly about this, but i'd probably lean multi-line. diffs are simpler to look at, if somehow we got an "unused feature" hint on the IDE it'd be easier to spot, too. but really, i don't care much either way.

i take it the formatter doesn't have an opinion?

Reply:
it does not!